XenModel/Actions/VM/ChangeMemorySettingsAction.cs

```diff
@@ -73,21 +74,9 @@ public class ChangeMemorySettingsAction : AsyncAction
             #endregion
         }
 
-        private bool staticChanged
-        {
-            get
-            {
-                return (static_min != VM.memory_static_min || static_max != VM.memory_static_max);
-            }
-        }
+        private bool staticChanged => static_min != VM.memory_static_min || static_max != VM.memory_static_max;
 
-        private bool needReboot
-        {
-            get
-            {
-                return (staticChanged && VM.power_state != vm_power_state.Halted);
-            }
-        }
+        private bool needReboot => staticChanged && VM.power_state != vm_power_state.Halted;

```

Comment:
+1 for using the nice short form for these properties while in the neighbourhood.